src/base/utils/foreignapps.cpp

```diff
@@ -190,9 +190,16 @@ namespace
                     path = getRegValue(hkInstallPath);
                     ::RegCloseKey(hkInstallPath);
 
-                    if (!path.isEmpty() && QDir(path).exists("python.exe")) {
-                        found = true;
-                        path = QDir(path).filePath("python.exe");
+                    if (!path.isEmpty()) {
+                        const QDir baseDir {path};
+
+                        if (baseDir.exists("python3.exe")) {
+                            found = true;
+                            path = baseDir.filePath("python3.exe");
+                        } else if (baseDir.exists("python.exe")) {
```

Comment:
This does not adhere to our coding style: `} else`, please follow the code suggestion closely.